docs/cli/commands.md

```diff
@@ -0,0 +1,561 @@
+  -h, --help        display help for command
+```
+
+## versions:bump
+
+Scope: `root`
+
+Bump all `@backstage` packages to the latest versions. This check for updates in
+the package registry, and will update entries both in `yarn.lock` and
+`package.json` files when necessary.
+
+```text
+Usage: backstage-cli versions:bump [options]
+
+Options:
+  -h, --help  display help for command
+```
+
+## versions:check
+
+Scope: `root`
+
+Validate `@backstage` dependencies within the repo, making sure that there are
+no duplicates of packages that might lead to breakages. For example,
+`@backstage/core` must not be loaded in twice, so having two different versions
+of it installed will cause this command to exit with an error.
+
+By supplying the `--fix` flag the command will attempt to fix any conflict that
+can be resolved by editing `yarn.lock`, but will not attempt to search for
+remote updates or modify any `package.json` files.
+
+```text
+Usage: backstage-cli versions:check [options]
+
+Options:
+  --fix       Fix any auto-fixable versioning problems
+  -h, --help  display help for command
+```
+
+## prepack
+
+Scope: `any`
+
+This command should be added as `scripts.prepack` in all packages. It enables
+packaging- and publish-time overrides for fields inside `packages.json`.
+
+The checked in version of all packages in a Backstage monorepo are tailored for
+local development, and as such `main` and similar fields inside `package.json`
+point to development source, i.e. `src/index.ts`. Using this when publishing
+would lead to a broken package, since `src/` is not included in the published
+package and we instead need to point to files in the `dist/` directory. This
+command allows for those fields to be rewritten when needed, and does so by
+copying all fields within `publishConfig` to the top-level of each
+`package.json`, skipping `access`, `registry`, and `tag`.
+
+The need for this command may be removed in the future, as this exact method of
+overriding fields for publishing is already supported by some package managers.
+
+```text
+Usage: backstage-cli prepack [options]
+
+Options:
+  -h, --help  display help for command
+```
+
+## postpack
+
+Scope: `any`
+
+This should be added as `scripts.postpack` in all packages. It restores
+`package.json` to what it looked like before calling the `prepack` command.
+
+```text
+Usage: backstage-cli postpack [options]
+
+Options:
+  -h, --help  display help for command
+```
+
+## clean
+
+Scope: `any`
+
+Remove cache and output directories.
+
+```text
+Usage: backstage-cli clean [options]
+
+Options:
+  -h, --help  display help for command
+```
+
+## build-workspace
+
+Scope: `any`, `root`
+
+Builds a mirror of the workspace using the packaged production version of each
+package. This essentially calls `yarn pack` in each included package and unpacks
+the resulting archive in the target `workspace-dir`.
+
+```text
+Usage: backstage-cli build-workspace [options] &lt;workspace-dir&gt;
+
+Options:
+```
```

Comment:
cut off

Reply:
It's the actual output of the command :grin:

Got rid of it tho :p